plasma_framework/contracts/src/framework/registries/VaultRegistry.sol

```diff
@@ -1,11 +1,16 @@
 pragma solidity ^0.5.0;
 
-import "../modifiers/Operated.sol";
+import "../utils/Operated.sol";
```

Comment:
I gave up the having modifiers folder because other modifier would quite likely to live in registries (`onlyFromVault` and `onlyFromExitGame`). Unless the modifier does not need to access to some variable. 

(previously tried to use "getter" interface to solve that but turn out would be around 2k more expensive on gas if access variable via `this.var()` instead of `var` directly)